wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -125,6 +125,19 @@ const TokenList = (props: Props) => {
       tokens.push(searchResult);
     }
 
+    // Add tokens that routes explicitly support, like token bridge, NTT
+    if (props.tokenList) {
+      for (const token of props.tokenList) {
+        if (
+          !isFrankensteinToken(token, props.selectedChainConfig.sdkName) &&
```

Comment:
filtering frankensteins here now, since the SDKv2 Route wrapped only filters on the source token (?) I think it's a bug but this adds a layer of protection on the output tokens